test/session_test.cr

```diff
@@ -40,7 +44,9 @@ class SessionTest < Minitest::Test
     count = types.size
 
     input = session.find_element(:css, "input[type='search']")
-    input.send_keys("Foo bar.., ups wrong...")
+    input.send_keys("Cleint.., ups wrong...")
+    sleep 1
```

Comment:
Any reason to introduce a sleep, here?

Reply:
umm, users can view posts before (and delay ...) -> .clear -> after